### PR TITLE
Typescript Typings

### PR DESCRIPTION
**Pull request by lucasriondel**
module definition for typescript usage.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export interface IRandomWordsOptions {
```
Comment (skinofstars) [inline index.d.ts]: Pretty sure you only need this `index.d.ts` file in the root, and that'd be enough. 

Any time you add another param, just make sure it's added to this list. The `?` in the param means it's optional.

Comment (lucasriondel) [reply index.d.ts:1]: Thank you for reviewing !

**Comment by lucasriondel** [comment]
@boutell Is it ok to merge ? :)

**Comment by boutell** [comment]
The branch has conflicts at this point, which will need to be addressed. I apologize for the delay.

**Pull request closed without merging**